mock/mock.spec

```diff
@@ -169,6 +169,8 @@ done
 
 ./precompile-bash-completion "mock.complete"
 
+%sysusers_create_compat mock.conf >> sysusers_script
```

Comment:
You need to call the shell script baked in `%sysusers_create_compat`, not the macro itself.